source/_integrations/zwave_js.markdown

```diff
@@ -460,6 +460,22 @@ In addition to the [standard automation trigger data](/docs/automation/templatin
 | `trigger.current_value`      | The current value for this Z-Wave value (translated to a state name when possible).        |
 | `trigger.current_value_raw`  | The raw current value for this Z-Wave value (the key of the state when a state is named).  |
 
+### `zwave_js_notification events`
```

Comment:
This is not the only event that the integration fires. The section `Automations` is about the custom trigger platforms that the zwave_js integration provides. We already have a section documenting the custom events of the integration: `zwave_js_notification`, `zwave_js_value_notification` etc. If you want to add an example it should probably go under the `Events` header.

The `Automations` header name is probably confusing and we should consider renaming it, separately.

Reply:
Makes sense, I've moved it to the events header.